src/probnum/filtsmooth/statespace/integrator.py

```diff
@@ -85,7 +83,7 @@ def _forcevec(self):
     @property
     def _dispmat(self):
         dispmat_1d = np.zeros(self.ordint + 1)
-        dispmat_1d[-1] = self.diffconst
+        dispmat_1d[-1] = 1.0  # Standard diffusion
```

Comment:
Should I remove the comment?

Reply:
No I think it is good, at least for the next few weeks/months until we got used to the interface.

Reply:
Though maybe call it unit diffusion instead of standard diffusion? Just a suggestion, feel free to ignore